### DISCUSSION TITLE
How can I limit the range of dynamic route matching?

### DISCUSSION THREAD

**Discussion by kecrily** [nuxt/nuxt#16914] [Questions]
If I have a dynamic route:

```
pages/[id].vue
```

I want to use it only when the `id` is `number`. Otherwise it should fall back to `pages/[...all].vue` or `pages/404.vue`.

**Comment by warflash** [comment]
You can't yet, I opened an issue for it a few weeks ago tho!
https://github.com/nuxt/framework/issues/6982

**Reply by kecrily** [reply]
My search keywords did not match this issue. Now I'm subscribed and looking forward to it.
